spec/models/manageiq/providers/amazon/regions_spec.rb

```diff
@@ -2,11 +2,20 @@
   it "has all the regions" do
     ems = FactoryBot.create(:ems_amazon_with_vcr_authentication)
 
-    # TODO: use `slice` on 2.5
-    current_regions = described_class.regions.select do |name, _config|
-      !described_class::SPECIAL_REGIONS.include?(name)
+    # https://github.com/aws/aws-sdk-ruby/blob/5fe5795e8910bb667996dfc75e4f16b7e69e3980/gems/aws-partitions/partitions.json#L11
+    ordinary_regions_regexp = /^(us|eu|ap|sa|ca)\-\w+\-\d+$/
+
+    # https://docs.aws.amazon.com/general/latest/gr/rande.html - see quotes below
+    atypical_regions = [
+      'ap-northeast-3', # "To request access to the Asia Pacific (Osaka-Local) Region, contact..."
+      'ap-east-1',      # "you must manually enable before you can use..."
+      'eu-south-1'
+    ]
```

Comment:
NOTE I didn't see anywhere other than this spec where `SPECIAL_REGIONS` or `ORDINARY_REGIONS_REGEXP` were used so I moved them here